src/test/java/ApplicationTest.java

```diff
@@ -0,0 +1,9 @@
+import juja.sqlcmd.Application;
```

Comment:
It looks like you have diferent structure of "main" and "test" folders... It is good practice this folders to have the same structure, so test classes can access (and test!) package-scope classes and members. Here's a good description of this point - http://bit.ly/2hyE8uO

Reply:
I never attached importance to this, but you are right. 
Thanks for link.
I'll changed structure of "test" folders.